core/store/src/lib.rs

```diff
@@ -733,6 +735,7 @@ impl StoreUpdate {
                 }
             }
         }
+        // TODO(reshardingV3) Map shard_uid for ops referencing State column.
```

Comment:
You added the mapping to the get method on the read path, why not add it to the set_* method on the write path?

Reply:
ignoring as of https://github.com/near/nearcore/pull/12084#issuecomment-2407167761